### PR TITLE
New version: acaudwell.Gource version gource-0.53

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Gource |     |
| Version     | gource-0.53     |  |
| Publisher   | Andrew Caudwell   |       |
| ProductCode | Gource, Gource          |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59199)

### CODE REVIEW

File: manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: acaudwell.Gource
+PackageVersion: gource-0.53
```
Comment (Trenly) [inline manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.installer.yaml:5]:
```suggestion
PackageVersion: 0.53
```

File: manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: acaudwell.Gource
+PackageVersion: gource-0.53
```
Comment (Trenly) [inline manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.53
```

File: manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+PackageLocale: en-US
+Publisher: Andrew Caudwell
```
Comment (Trenly) [inline manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.locale.en-US.yaml:7]:
```suggestion
Publisher: acaudwell
```

File: manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: acaudwell.Gource
+PackageVersion: gource-0.53
```
Comment (Trenly) [inline manifests/a/acaudwell/Gource/gource-0.53/acaudwell.Gource.yaml:5]:
```suggestion
PackageVersion: 0.53
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — CHANGES_REQUESTED** [review]
I'll let Vedant fix the version number, since it should be applied in automation

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**